modules/ROOT/pages/faq.adoc

```diff
@@ -336,3 +336,44 @@ systemd:
 
 For more information see
 https://github.com/coreos/fedora-coreos-tracker/issues/519[the tracker issue discussion].
+
+== Why does SSH stop working after upgrading to Fedora 33?
+
+In Fedora 33 there was a change to
+https://www.fedoraproject.org/wiki/Changes/StrongCryptoSettings2[implement stronger crypto defaults].
+Part of this included taking the
+https://www.openssh.com/txt/release-8.3[advice of OpenSSH] upstream
+and disabling the use of the `ssh-rsa` public key signature algorithm.
+
+You may hit issues if you use RSA keys and:
+
+* use an old version of the `SSH` client
+* use tooling/software libraries that don't support using RSA SHA2 host key signatures
```

Comment:
"public key signatures" would be clearer, since host keys aren't involved here.

Reply:
thanks for the suggestion. Fix in #222